<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Kalpesh" name="Default test" tests="19" failures="1" timestamp="28 Oct 2017 00:53:09 GMT" time="186.251" errors="0">
  <testcase name="submitFormTest" time="32.476" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="7.699" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="8.348" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="7.383" classname="com.test.FormTest">
    <failure type="java.lang.AssertionError" message="expected [The max length of last name is 20] but found [The max length of first name is 20]">
      <![CDATA[java.lang.AssertionError: expected [The max length of last name is 20] but found [The max length of first name is 20]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.test.FormTest.submitFormTest(FormTest.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- submitFormTest -->
  <testcase name="submitFormTest" time="6.511" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="7.04" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="6.999" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="7.769" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="7.324" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="6.947" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="7.395" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="7.275" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="6.487" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="5.958" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="5.865" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="8.229" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="8.657" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="6.845" classname="com.test.FormTest"/>
  <testcase name="submitFormTest" time="7.279" classname="com.test.FormTest"/>
</testsuite> <!-- Default test -->
